index.html

```diff
@@ -10,39 +10,250 @@
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+         
+          <button class="see-project">See project</button>
+      </section>
+
+      <section class="work">
+        <img
+           width="350px"
+          src="./image/work2.png"
+        />
+        <h3 class="work-title">MUlti-Post Stories</h3>
+        <div class="profession">
+          <p>CANOPY</p>
+          <div class="expert">
+            <div class="bullet"></div>
+            <p>Back End Dev</p>
+          </div>
+          <div class="expert">
+            <div class="bullet"></div>
+            <p>2015</p>
+          </div>
+        </div>
+        <h3 class="description">
+          A daily selection of privately personalized reads;no accounts or
+          sign-ups required
+        </h3>
+
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+         
+          <button class="see-project">See project</button>
+      </section>
+      <section class="work">
+        <img
+           width="350px"
+          src="./image/work3.png"
+        />
+        <h3 class="work-title">Tonic</h3>
+        <div class="profession">
+          <p>CANOPY</p>
+          <div class="expert">
+            <div class="bullet"></div>
+            <p>Back End Dev</p>
+          </div>
+          <div class="expert">
+            <div class="bullet"></div>
+            <p>2015</p>
+          </div>
+        </div>
+        <h3 class="description">
+          A daily selection of privately personalized reads;no accounts or
+          sign-ups required
+        </h3>
+
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+         
+          <button class="see-project">See project</button>
+          </section>
+          <section class="work">
+            <img
+               width="350px"
+              src="./image/work4.png"
+            />
+            <h3 class="work-title">MUlti-Post Stories</h3>
+            <div class="profession">
+              <p>CANOPY</p>
+              <div class="expert">
+                <div class="bullet"></div>
+                <p>Back End Dev</p>
+              </div>
+              <div class="expert">
+                <div class="bullet"></div>
+                <p>2015</p>
+              </div>
+            </div>
+            <h3 class="description">
+              A daily selection of privately personalized reads;no accounts or
+              sign-ups required
+            </h3>
+    
+              <ul class="languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
+             
+              <button class="see-project">See project</button>
+              </section>
+              <section class="work">
+<h3 class="work-title">About my self</h3>
```

Comment:
- [ ] Please try your best to change the color, and size of the about section title. This will help us to match the Figma design and also make sure that our portifolio has a good UI ☺️.
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/236160780-4dcf3e83-a5c7-4e29-8271-6ab4549515fd.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/236160867-381bd0a0-bce8-45a8-ad09-dc84b03361dc.png)